README.rst

```diff
@@ -53,7 +56,4 @@ Cite
 
 If you use HyperSpy in a scientific publication, please cite the following DOI:
```

Comment:
Either this sentence is removed or the DOI is also included after but it can't be in between! ;)

Reply:
Ah you reviewed it at the same time I hit merge :-/

Reply:
Thanks done